assets/js/base/context/providers/cart-checkout/checkout-processor.ts

```diff
@@ -293,12 +295,20 @@ const CheckoutProcessor = () => {
 							__internalProcessCheckoutResponse( response );
 						} );
 				} catch {
+					let errorMessage = __(
+						'Something went wrong, but your order may still have been placed. Please check your email inbox before retrying.',
```

Comment:
`Something went wrong when placing the order. Check your email for order updates before retrying.`